src/components/main/navbar/index.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import styles from '../../../css-modules/main.module.css';
+
+// Prop Types
+import PropTypes from 'prop-types';
+
+const NavBar = ({ children, typeSelected, setTypeSelected, sortBySelected, setSortBySelected }) => {
+
+    const changeItemSelected = (type) => {
+        setTypeSelected(type)
+    }
+
+    return (
+        <section>
```

Comment:
I guess that navbar should contain only nav elements. So you don't need to use section and children here

Reply:
Resolved